corehq/apps/callcenter/indicator_sets.py

```diff
@@ -381,11 +374,16 @@ def add_cases_active_data(self, range_name, lower, upper):
         cases_active_{period}
         cases_active_{case_type}_{period}
         """
-        results = self._base_case_query_coalesce_owner() \
+        results = CaseActionData.objects \
+            .extra(select={'type': 'case_type'}) \
+            .values('case_owner', 'type') \
+            .exclude(case_type=self.cc_case_type) \
             .filter(
-                actions__date__gte=lower,
-                actions__date__lt=upper
-            ).annotate(count=Count('case_id', distinct=True))
+                domain=self.domain,
+                case_owner__in=self.owners_needing_data,
+                date__gte=lower,
+                date__lt=upper
+            ).annotate(count=Count('case', distinct=True))
```

Comment:
Not really worth trying to share the params I don't think. It also makes it harder to figure out what the query is.